legate/driver/launcher.py

```diff
@@ -51,6 +44,14 @@
 )
 
 
+RANK_ERR_MSG = """\
+Could not detect rank ID on multi-rank run with no --launcher provided. If you
+want Legate to use a launcher, e.g. mpirun, internally (recommended), then you
+need to specify which one to use by passing --launcher. Otherwise you need to
+invoke the legate script itself through a launcher.
```

Comment:
Should the message be softened or changed? It now happens if `LEGATE_RANK` is not set, for any reason.

Reply:
This message is currently gone since the global rank is replaced by `bind.sh` in all cases (it will error if rank cannot be determined)